numpy/distutils/misc_util.py

```diff
@@ -2326,6 +2326,43 @@ def get_info(name):
                 return g.get(name, g.get(name + "_info", {}))
 
             def show():
+                """
+                Print information about BLAS and LAPACK libraries in use.
+
+                Print information about various resources (libraries, library
+                directories, include directories, etc.) in the system on which
+                NumPy was built.
+
+                See Also
+                --------
+                get_include : Returns the directory containing NumPy C
+                              header files.
+
+                Notes
+                -----
+                Classes specifying the information to be printed are defined
+                in the `numpy.distutils.system_info` module.
+
+                Information may include:
+
+                * ``language``: language used to write the libraries (mostly
+                  C or f77)
+                * ``libraries``: names of libraries found in the system
+                * ``library_dirs``: directories containing the libraries
+                * ``include_dirs``: directories containing library header files
+                * ``src_dirs``: directories containing library source files
+                * ``define_macros``: preprocessor macros used by
+                  ``distutils.setup``
+
+                Examples
+                --------
+                >>> np.show_config()
```

Comment:
This is not the function that this docstring is attached to.

Reply:
`show_config` is the new name given to `show` when it is imported into the `numpy` namespace (in `numpy/__init__.py`).  I don't know why it isn't called `show_config` in `__config__.py`.

Reply:
Isn't `np.show_config == np.__config__.show`? This file is not `np.__config__`.